driver/normalizer/annotation.go

```diff
@@ -28,6 +28,8 @@ var Native = Transformers([][]Transformer{
 	},
 }...)
 
+var Code []CodeTransformer // TODO(bzz): deprecated
```

Comment:
This was removed during SDK update as deprecated.

Reply:
Thank you for spotting! A rebase went rogue...